packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.component.js

```diff
@@ -0,0 +1,26 @@
+import controller from './hosting-cdn-shared-settings.controller';
+import template from './hosting-cdn-shared-settings.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    autoPayWithPreferredPaymentMethod: '<',
+    catalogAddon: '<',
+    checkoutOrderCart: '<',
+    goBack: '<',
+    goBackWithError: '<',
+    isOptionFree: '<',
+    prepareOrderCart: '<',
+    serviceName: '<',
+    serviceOption: '<',
+    user: '<',
+    hasCDN: '<',
+    isV1CDN: '<',
+    isIncludedCDN: '<',
+    isPayableCDN: '<',
+    isV2CDN: '<',
+    cdnProperties: '<',
+  },
+  controllerAs: '$HostingSharedCdnCtrl'
```

Comment:
```suggestion
```

The default value is `$ctrl` and we prefer to use it.

Reply:
changed